extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -436,12 +452,13 @@ export class AmpConsent extends AMP.BaseElement {
    * @param {?JsonObject} response
    */
   parseConsentResponse_(instanceId, response) {
-    if (!response || !response['promptIfUnknown']) {
-      //Do not need to block.
-      this.consentRequired_[instanceId] = false;
-    } else {
-      // TODO: Check for current consent state and decide if UI is required.
+    if (response && response['promptIfUnknown'] == true) {
       this.consentRequired_[instanceId] = true;
+    } else if (response && response['promptIfUnknown'] == false) {
+      this.consentRequired_[instanceId] = false;
```

Comment:
let's avoid writing this field in multiple places. it will be error-prone later on.